packages/ember-data/lib/system/model/model.js

```diff
@@ -33,6 +33,11 @@ var Model = Ember.Object.extend(Ember.Evented, {
 
   store: null,
 
+  // Remove support for `record.container`
+  get container() {
+    Ember.assert("A model's private reference to the container has been removed for consistency reasons. Instances of a store (i.e. `this.store`) have access to the container, although usage from within a model is discouraged and can lead to unintended consequences.");
+  },
+
```

Comment:
You should create the getter dynamically inside an Ember.runInDebug so it gets stripped in production. However apparently phantom 1.9 doesn't implement defineProperty, but 2.0 does so @fivetanley is the person to ask about that